CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ FEATURES:
 * Add support for configuration of plugin WIF to the AWS Secret Backend. Requires Vault 1.16+ ([#2138](https://github.com/hashicorp/terraform-provider-vault/pull/2138)).
 * Add support for Oracle database plugin configuration options `split_statements` and `disconnect_sessions`: ([#2085](https://github.com/hashicorp/terraform-provider-vault/pull/2085))
 
+IMPROVEMENTS:
```

Comment:
Or BUG? I went with improvement since I have not been able to verify that this fixes the reported error.